example/pages/index.tsx

```diff
@@ -43,6 +43,11 @@ const Home: NextPage = () => {
     [provider]
   )
 
+  const redirect = useCallback(
+    (address) => window.open(`https://etherscan.io/address/${address}`),
+    []
+  )
```

Comment:
Instead of redirecting, let me pass `router.push(action)` to `onUserClick`
The client needed to navigate to user on their platform not etherscan

Reply:
This is just for the test app https://chat-liteflow.vercel.app/ (or the preview deployed for test).
This test app does not have any profile so in that case etherscan can work but yes indeed on a real application then this will probably use the router to navigate inside the app